build.sbt

```diff
@@ -1,19 +1,11 @@
-
-inThisBuild(
-  (homepage := Some(url("https://github.com/gemini-hlsw/sbt-lucuma"))) +: lucumaPublishSettings
-)
+ThisBuild / tlBaseVersion := "0.5"
```

Comment:
What's `tlBaseVersion`?

Reply:
It's part of the versioning scheme inherited from sbt-spiewak. Basically if your current version is `x.y.z` then your `tlBaseVersion` is `x.y.`. When you start working on your next version, you bump it.

https://github.com/djspiewak/sbt-spiewak/blob/main/versioning.md#compatibility-version
https://typelevel.org/sbt-typelevel/#how-do-i-introduce-breaking-changes-intended-for-my-next-version

Reply:
I think sbt-lucuma is 0.4.4 now. Should this be `0.4`?

Reply:
I think the changes here constitute breaking changes -> `0.5`. WDYT?

Reply:
I see